pkg/bloombuild/builder/builder_test.go

```diff
@@ -99,41 +126,56 @@ func Test_BuilderLoop(t *testing.T) {
 
 type fakePlannerServer struct {
 	tasks          []*protos.ProtoTask
-	completedTasks atomic.Int64
+	completedTasks int
```

Comment:
Is the data race a false positive?

```
WARNING: DATA RACE
Read at 0x00c00073dc58 by goroutine 79:
  github.com/grafana/loki/v3/pkg/bloombuild/builder.(*fakePlannerServer).CompletedTasks()
      /home/christian/sandbox/grafana/loki/pkg/bloombuild/builder/builder_test.go:204 +0xd2c
```

Reply:
I think it can actually happen. Changed back to use the atomic.